.gitignore

```diff
@@ -25,3 +25,5 @@ tests/unit/coverage/
 *.njsproj
 *.sln
 *.sw?
+
+app/*
```

Comment:
I don't think it is necessary to modify the .gitignore file for this PR. Please revert.

```suggestion
```